web/web.go

```diff
@@ -35,7 +35,7 @@ import (
 // Commandline flags.
 var (
 	listenAddress  = flag.String("web.listen-address", ":9090", "Address to listen on for the web interface, API, and telemetry.")
-	metricsPath    = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
+	metricsPath    = flag.String("web.telemetry-path", "metrics", "Path under which to expose metrics.")
```

Comment:
Good point. Yes, this should still start with a slash to be consistent with the other servers.